{{cookiecutter.project_slug}}/README.md

```diff
@@ -91,11 +91,34 @@ When jobs with HTTP interface are executed (for example, with Jupyter Notebooks
 a user to be authenticated on the platform. However, if you want to share the link with someone who is not registered on
 the platform, you may disable the authentication updating this line to `HTTP_AUTH?=--no-http-auth`.
 
-### Training command
-
-`TRAINING_COMMAND?='echo "Replace this placeholder with a training script execution"'`
-
-If you want to train some models from code instead of Jupyter Notebooks, you need to update this line. For example:
-
-`TRAINING_COMMAND="bash -c 'cd $(PROJECT_PATH_ENV) && python -u $(CODE_DIR)/train.py --data $(DATA_DIR)'"`
-
+### Training
+
+To run a training job, please first set up your own training command in environment variable `TRAINING_COMMAND` in either of the following ways:
+1. Set up env variable for current shell:
+```
+export TRAINING_COMMAND="python /path/to/script.py"
+make training
+```
+
+2. Set up env variable for a single run:
+```
+make training  TRAINING_COMMAND="python /path/to/script.py"
+```
+
+3. Set up env variable permanently: open file `./Makefile` and replace
+```
+TRAINING_COMMAND?='echo "Replace this placeholder with a training script execution"'
+```
+with
+```
+TRAINING_COMMAND="python /path/to/script.py"
+```
+And then, just run `make training`
+
+
+You can also run multiple training experiments simultaneously. For that, set up `EXPERIMENT` environment variable:
+```
+make training  EXPERIMENT=check-new-idea
+```
+Please note that this label becomes a postfix of the job name, which may contain only alphanumeric characters 
```

Comment:
Please, add information on how to kill a job with postfix in a name. It's unclear now that you need to use the same postfix to kill a training job.

Reply:
right! thank you!

Reply:
fixed